misk-embedded/src/test/kotlin/misk/embedded/EmbeddedMiskTest.kt

```diff
@@ -0,0 +1,67 @@
+package misk.embedded
+
+import misk.embedded.sample.embedded.EmbeddedSampleService
+import misk.embedded.sample.embedded.SampleEvent
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class EmbeddedMiskTest {
```

Comment:
It would be nice to see what a service to service action invocation looks like when it's from one test service to an embedded test service. I'm wondering if we're going to end up using test HTTP clients and exercising our stack of request interceptors, or just bypass that with direct invocations to action classes in the embedded service's dependency graph.

Reply:
I hope we can make direct invocations work. Step debugging is great!

(And if we need interceptors for stuff like auth, we don’t need HTTP for that. We can use decorator patterns.)